### DISCUSSION TITLE
Enhancement/Alteration: Change Mobius Stat Penalty and Intensity Behavior

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#18047] [Feature requests] [answered]
Currently, the game enforces Penalties unfairly when a coaster is multistation
![73Land 2022-09-18 03-52-31](https://user-images.githubusercontent.com/94884086/190892233-31217622-ec3d-425a-93a3-f6a454b947c2.png)
this is how I had to alter this coaster to get decent stats
![73Land 2022-09-18 04-03-03](https://user-images.githubusercontent.com/94884086/190892254-dc6dc6e5-a718-43dd-9c5f-1da23627bdfd.png)

my proposal is to alter stat calculation so that on a multistation coaster, each segment is treated with penalties and intensity separately. So if ride segment 3 has high lateral g's that only counts for 1/3 of the penalty it normally would AND if a ride has 12 loops on segment 2, it would still be extreme intensity BUT it would not get ultra extreme if it's spread out between all the segments

I am guessing to implement this the code would have to have some way of identifying the number of stations, then when a penalty is thrown ie too many hills or too many inversions, the intensity calculation would be determined after factoring in the number of stations BUT it wouldn't just take the average

I don't want it to simply divide by the number of stations because people would abuse it, I want it to actually calculate all the segments of the ride as separate rides then average that calculation to see if it deserves a penalty or an extreme or ultra extreme intensity

**Comment by ocalhoun6** [comment]
Any coaster with 11 inversions is going to have a lot of intensity. (Not to mention 6.61 positive vertical G's -- a lot of real-world guests would pass out from that much!) The first 6 inversions add both intensity and excitement, but any more than 6 will add intensity only and no excitement.

The different number of drops is also a factor. Each drop also adds intensity. And 16 drops is getting pretty high.

-------

I'm pretty sure the number of stations has nothing to do with this, and that if you rebuilt both of these rides with only 1 station (and brakes to simulate the slowdown of a station) then they'd have about the same stats anyway.

The game's not penalizing your coaster because it has multiple stations ... it's just penalizing your coaster because it's overly huge and has too many elements that add intensity (especially inversions).

**Comment by 733737** [comment, marked as answer]
that's not true, each segment has an entrance and exit, the peeps do not go through more than 3 inversions. Twister coasters can have 6 inversions and only receive 6 excitement and 6 intensity

**Reply by ocalhoun6** [reply]
It doesn't matter whether the peeps go through the inversions or not. The stats are calculated based on the entire ride.

You could have an exit-only station, then a bunch of inversions, then an entrance-only station that takes peeps back to the first station, so that no peeps ever ride the inversions ... and the inversions will still be counted in the ride stats.

**Reply by 733737** [reply]
that's kind of the whole point of the discussion, to not treat the entire ride as one ride for the purple of intensity and penalty calculation. This would also mean that rides that have segments that are very boring would not attract guests, so it would balance itself. I think that you aren't viewing this the same way as me, not because you disagree, but because you misunderstand what I want. I just want to be able to build a mobius coaster that acts like multiple coasters when calculating stats

**Reply by ocalhoun6** [reply]
Oh _that's_ what you're trying to get at?

Already have a request for that here: https://github.com/OpenRCT2/OpenRCT2/discussions/15962

**Reply by 733737** [reply]
oops my bad i guess it's a duplicate then
